make-release.sh

```diff
@@ -8,16 +8,10 @@ ORGANIZATION="eclipse"
 
  # KEEP RIGHT ORDER!!!
 DOCKER_FILES_LOCATIONS=(
-    che-server/dockerfiles/endpoint-watcher
-    che-server/dockerfiles/keycloak
-    che-server/dockerfiles/postgres
```

Comment:
Since you removed `che-server/dockerfiles/che` you probably want to change that to `build/dockerfiles` but note also line 309/303 https://github.com/eclipse-che/che-server/pull/573/files#diff-be36842270d47fd0f7dd297abbdf5c5ef7916efe78fd4617f5ca6a5fdcba421cL309-R303 might need to change too, so it points correctly to where the `build.sh` script is now located at `build/build.sh` rather than in teh same folder as the dockerfile.